### PR TITLE
Suppress debug-level logs when required

### LINKED ISSUE #86: Suppress logging
Is there a way to suppress debug logging from the library, or control it somehow? It's quite annoying to see all the logs printed.

### PR DESCRIPTION
**Pull request by ItalyPaleAle**
Fixes #86

### CODE REVIEW

**Comment by denisvmedia** [comment]
You can actually control the logs without modifying the library. Just change the log output:

```go
log.SetOutput(ioutil.Discard)
```

**Comment by ItalyPaleAle** [comment]
My PR is a bit different because it doesn’t suppress *all* logs, but just debug ones. I still want to see errors

**Comment by ItalyPaleAle** [comment]
@denisvmedia Following up on this... Any chance you could merge this?

Your workaround suppresses all logs, for the entire application. It operates in the global scope. I'd love it to just suppress the logs for govips and nothing else, and especially just have control over debug logs only.

As you can see between here and the other issue, there are upvotes by other too

**Comment by denisvmedia** [comment]
Sorry to disapoint you, but no chances. I'm not the owner of the repo ;)

**Comment by ItalyPaleAle** [comment]
That was.... awkward 😄  Sorry!!

@davidbyttow You can probably merge this!

File: pkg/vips/govips.go [hunk 1]
```diff
@@ -29,6 +29,9 @@ const VipsMajorVersion = int(C.VIPS_MAJOR_VERSION)
 // VipsMinorVersion if the minor version of libvips
 const VipsMinorVersion = int(C.VIPS_MINOR_VERSION)
 
+// ShowDebugLog controls whether debug-level log is printed
+var ShowDebugLog = true
```
Comment (davidbyttow) [inline pkg/vips/govips.go:33]: This should probably default to false

**Comment by davidbyttow** [comment]
But will merge as it doesn't change behavior.

**Pull request merged**